personalized_nlp/datasets/datamodule_base.py

```diff
@@ -368,13 +379,19 @@ def _compute_annotator_stats(self):
         annotations_with_data = self.annotations.merge(self.data[data_cols])
 
         if self.stratify_folds_by == "users":
+            annotations_with_data = self.annotations.merge(
```

Comment:
To chyba jest powtórzenie kodu z linijki 368?

Reply:
To prawda - warto się zastanowić czy nie warto by było usunąć kodu z linijki 368, gdyż w linijce `387` mamy nadpisanie tej zmiennej. Oznacza to, że jeżeli warunek z linijki `370` nie jest spełniony to operacja `merge` z linijki `368` jest redundantna.

Reply:
Ok, można w takim razie usunąć to z 368

Reply:
Usunąłem linijkę `368` oraz uprościłem wykorzystanie listy `data_cols` przenosząc dodanie nazwy jednej kolumny z warunku z linijki `365` do tego samego warunku w linijce `370`.